server/queue/queue-rabbitmq/src/main/java/org/apache/james/queue/rabbitmq/view/cassandra/CassandraMailQueueMailDelete.java

```diff
@@ -66,15 +68,18 @@
         return deletedMailsDao.isDeleted(mailQueueName, MailKey.fromMail(mail));
     }
 
-    void updateBrowseStart(MailQueueName mailQueueName) {
+    CompletableFuture<Void> updateBrowseStart(MailQueueName mailQueueName) {
         Mono<Instant> newBrowseStart = findNewBrowseStart(mailQueueName);
-        updateNewBrowseStart(mailQueueName, newBrowseStart);
+        return updateNewBrowseStart(mailQueueName, newBrowseStart)
```

Comment:
no need to turn it to a Future.
```java
        updateNewBrowseStart(mailQueueName, newBrowseStart)
            .subscribeOn(Schedulers.elastic())
            .subscribe()
```

Reply:
3 line in both cases. What's the difference?

Reply:
Cool thanks